Rakefile

```diff
@@ -46,9 +46,11 @@ end
 desc 'Grab libs'
 task 'setup_libs' do
   in_venv = system "python -c \"import sys ; exit(not hasattr(sys, 'real_prefix'))\""
-  raise 'Not in dev venv/CI environment - bailing out.' if !in_venv && !ENV['CI']
+  raise 'Not in dev venv/CI environment/Integrations - bailing out.' if !in_venv && !ENV['CI'] && !ENV['SDK_HOME']
```

Comment:
I don't think it would break it? Unless I misunderstand the logic there, it's saying, raise this if none of these things are present. But if it is present, then it wouldn't raise. So, if it's in the venv, then it's fine.